src/styles/modules/_TextEditor.scss

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
We might have to decide whether to separate these or unify them. Also let's streamline our name case conventions here